src/ui/common/src/components/tables/CheckTableItem.tsx

```diff
@@ -1,71 +1,31 @@
-import {
-  faCircleCheck,
-  faCircleExclamation,
-  faMinus,
-  faTriangleExclamation,
-} from '@fortawesome/free-solid-svg-icons';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import Box from '@mui/material/Box';
 import React from 'react';
 
-import { theme } from '../../styles/theme/theme';
-import { stringToExecutionStatus } from '../../utils/shared';
+import ExecutionStatus from '../../utils/shared';
 import { StatusIndicator } from '../workflows/workflowStatus';
 
 interface CheckTableItemProps {
-  checkValue: string;
-  status?: string;
+  status: ExecutionStatus;
+  value?: string;
 }
 
 export const CheckTableItem: React.FC<CheckTableItemProps> = ({
-  checkValue,
   status,
+  value,
 }) => {
-  let iconColor = theme.palette.black;
-  let checkIcon = faMinus;
+  if (value) {
+    return <>{value}</>;
```

Comment:
Does this need to be a Typography component? Or does the parent component take care of this?

Reply:
This follows what has been used in main version that we returns the raw string for check names, etc.

Reply:
We could use a Typography here but I feel that's an independent change and not necessarily need to spend much time in this PR

Reply:
no worries, just checking what's going on here.